src/sumo.erl

```diff
@@ -38,13 +38,12 @@
 %%% API for standard CRUD functions.
 -export([
   persist/2,
+  fetch/2,
+  find_all/1, find_all/4,
+  find_by/2, find_by/4, find_by/5,
   delete/2,
   delete_by/2,
   delete_all/1,
-  find/2,
-  find_all/1, find_all/4,
-  find_by/2, find_by/4, find_by/5,
-  find_one/2,
```

Comment:
You shouldn't remove this function. Sometimes you want to find the first element that matches certain conditions, e.g. an alternative unique index.